projects/inih/parse_fuzzer.cc

```diff
@@ -0,0 +1,22 @@
+#include <cstdlib>
+#include <string>
+#include "ini.h"
+
+int dumper(void* user, const char* section, const char* name,
+           const char* value)
+{
+    return 1;
+}
+
+void parse(const char* string) 
+{
+    int u = 100;
+    int e = ini_parse_string(string, dumper, &u);
+}
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+    std::string str((const char*)data, size);
```

Comment:
can you please use reinterpret_cast instead?

Reply:
Done